copier/config/factory.py

```diff
@@ -84,9 +84,12 @@ def make_config(
     config_data, questions_data = filter_config(file_data)
     config_data.update(_metadata)
     del _metadata
-    config_data["data"] = query_user_data(questions_data, answers_data, not force)
     args = {k: v for k, v in locals().items() if v is not None and v != []}
+    env_ops = EnvOps(**config_data.get("envops", {}))
+    config_data["data"] = query_user_data(
+        questions_data, answers_data, not force, env_ops.dict()
```

Comment:
IMHO you better pass directly the `EnvOps` to benefit from typing.

```suggestion
        questions_data, answers_data, not force, env_ops
```

Reply:
Done.